### DISCUSSION TITLE
Code example

### DISCUSSION THREAD

**Discussion by kehiy** [libp2p/rust-libp2p#4446] [Q&A] [answered]
hello
Is there any simple project with `rust-libp2p`? (except the repository example file)
If you know one, I will be happy to have the link!

**Comment by mxinden** [comment, marked as answer]
You might find https://github.com/mxinden/libp2p-lookup/ helpful.

**Reply by kehiy** [reply]
thank you so much!

**Reply by thomaseizinger** [reply]
There is also https://github.com/libp2p/rust-libp2p/tree/master/misc/server.

**Reply by kehiy** [reply]
thank you so much tooo
